models/sync-media/README.md

```diff
@@ -0,0 +1,96 @@
+# Synchronized Media
+
+This document introduces a new JSON based syntax for synchronizing media together.
+
+## Listing documents
+
+
+```json
+"syncMedia": [
+  {
+    "href": "sync.json"
+  },
+  {
+    "href": "sync2.json"
+  }
+]
+```
+
+## Model
+
+The new syntax is based around a single JSON object called a Sync Media Object:
+
+| Name | Description | Format |
+| ---- | ----------- | ------ |
+| `alt` | Text alternative for the current Sync Media Object. | URI |
+| `audioref` | Points to a media fragment in an audio resource. | URI |
+| `children` | Array of Sync Media Objects. | Sync Media Objects |
+| `imgref` | Points to a media fragment in an image resource. | URI |
+| `role`     | Array of roles relevant for the current Sync Media Object. | Array of roles |
+| `text`  | Text equivalent for the current Sync Media Object. | String |
+| `textref`  | Points to a media fragment in an HTML/XHTML resource. | URI |
+
+Each Sync Media Object MUST contain:
+
+- a `children` object containg at least one Sync Media Object
+- or at least two of the following elements: `audioref`, `imgref`, `text` or `textref`
+
+## Roles
+
+- <https://www.w3.org/TR/epub-ssv-11/>
+
+
+
+## Examples
+
+*Example 1: Simple Sync Media Object with text and audio synchronized together*
+
+```json
+{
+  "textref": "chapter1.html#par1",
+  "audioref": "chapter1.mp3#t=0,20"
+}
+```
+
+*Example 2: Text equivalent of a speech bubble in a comic book*
+
+```json
+{
+  "role": ["panel"],
+  "imgref": "page10.jpg#xywh=percent:10,10,60,40",
+  "alt": "A cowboy is looking at the city as the sun sets into the horizon."
+  "children": [
+    {
+      "role": ["balloon"]
+      "imgref": "page10.jpg#xywh=percent:10,10,20,20",
```

Comment:
Speech balloons are usually complex shaped in my experience. Could we find a way to provide a clip-path for the balloon?

Something like
```json
{
...
              "clip-path": "polygon(38.47%% 4.47%%,38.43%% 6.61%%,20.32%% 6.53%%,20.36%% 4.38%%)",
...
}
```

Reply:
Ideally, this would be represented using a media fragment. We could explore what the IDPF proposed in region based navigation: https://idpf.org/epub/renditions/region-nav/#sec-3.5.1.3

Reply:
That's great, I didn't know that existed.